apps/artist/queries/server.coffee

```diff
@@ -18,6 +18,9 @@ module.exports = """
         text
         credit
       }
+      _artworks: artworks(size: 50, filter: IS_FOR_SALE) {
+        _id
+      }
```

Comment:
@craigspaeth I'm pretty sure this is bad but I don't know exactly how it needs to change. It currently makes a huge request for artworks from Force's server side before rendering _any_ page under /artist/. I can't tell if/how the data is used.

Can we drop this? If not, can we aggressively cache the results to avoid the unnecessary request?